Develop

mikethreels opened the pull request:
Changelog

- Added top navbar
- Added main paragraph section
- Added pictures
- Added video section
- Added aside section
- Added footer section

mikethreels committed:
setup configuration files

mikethreels committed:
add navigation bar in html

mikethreels committed:
navbar style edit

mikethreels committed:
positioning nav-bar buttons

mikethreels committed:
cleaned up styles

mikethreels committed:
changed logo picture

mimipeshy committed:
fixed linter errors

mimipeshy committed:
Merge pull request #1 from mikethreels/ft-top-nav

Ft-to-nav

mimipeshy committed:
made changes on the main nav

mimipeshy committed:
updated fonts and updated css files

mimipeshy committed:
fixed header section css styling

mikethreels committed:
added paragraphs up until the second picture

mikethreels committed:
finished styling the second picture

mikethreels committed:
finished main section

mikethreels committed:
added headers for aside section

mikethreels committed:
downloaded the pictures for aside section

mimipeshy committed:
Merge pull request #3 from mikethreels/ft-main-nav

Ft main nav

mimipeshy committed:
aside section

mikethreels committed:
fixed grid in aside section

mimipeshy committed:
styled aside section

mikethreels committed:
changed the aside section to grid

mikethreels committed:
updated the most popular section

mikethreels committed:
final adjustments inside the aside section

mikethreels committed:
deleted commented code

mikethreels committed:
Merge pull request #5 from mikethreels/ft-alt-aside

Ft alt aside

mimipeshy committed:
merged footer section with aside section

mikethreels committed:
added first part of footer

mikethreels committed:
fixed merge conflict

mikethreels committed:
fixed merge conflicts

mikethreels committed:
Merge pull request #6 from mikethreels/ft-alt-aside

fixed merge conflicts

mimipeshy committed:
finished footer section

mimipeshy committed:
fixed stylelint errors

mimipeshy committed:
Merge pull request #7 from mikethreels/ft-footer

finished footer section

mikethreels committed:
updated readme

mikethreels committed:
cleaned up code and fix bugs

mikethreels committed:
final bug fixes

mikethreels committed:
Merge pull request #8 from mikethreels/ch-fix-bugs

Ch fix bugs

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

addod19 commented on index.html:
```diff
@@ -0,0 +1,675 @@
+                    </div>
+                </a>
+                <a href="#">
+                    <div class="topic-description">
+                        <img src="images/analog-clock.jpg" alt="Analog clock">
+                        <div class="mini-title-container">
+                            <span class="mini-title washed-out flex-end">Paul Kooiker for The New York Times</span>
+                        </div>
+                        <h3 class="main-title customfont">How Analog Clocks Can Give Us More by Giving Us Less</h3>
+                        <span class="date-posted">May 5</span>
+                    </div>
```
We could improve semantics by using ```<article>``` instead of ```< div class="topic-description">```

addod19 reviewed (changes requested):
## Status: Require changes :recycle: :traffic_light: 

Hi, @mikethreels and @mimipeshy, great job on this project :+1: congratulations. Let us improve our code and make it better. Kindly find comments below: :smile: 

- The text should have a link, currently, it does not
![last](https://user-images.githubusercontent.com/46388113/81619056-4058c580-93d8-11ea-837d-bda670a64d66.png)
- Kindly add a detailed description to the README. You can check pathwright for the description, for example 'use float, flex, and grid to position elements.
- The live link is currently not working properly, kindly add more social handles to the author section
- Let us apply the DRY rule to our CSS file, so we avoid repeating code. This [article](https://speckyboy.com/good-bad-css-practices/)

KIndly make these changes and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Larbi](https://speckyboy.com/good-bad-css-practices/)

mimipeshy committed:
worked on TSE Review

mikethreels committed:
clean CSS code

mikethreels committed:
final CSS and HTML bug fixes

mikethreels committed:
Merge pull request #11 from mikethreels/gh-pages

gh-pages

mikethreels commented:
Changelog
- updated hamburger button, search icon and the text in the navbar.
- updated more detailed README file.
- fixed the live link.
- cleaned up the CSS.
- replaced the `div class="topic-description"` to `article`

lucasmazo32 reviewed (approved):
# Project approved :rocket:

#### Great job on implementing the changes!

#### Your project looks great!

### You can go ahead and merge this pull request!

### Happy coding! :computer:

## Lucas Mazo
